virt/virt-4-13-release-notes.adoc

```diff
@@ -88,6 +88,7 @@ The SVVP Certification applies to:
 //CNV-22555 Release note: CHANGE Hostname field moving from cloud-init options to VM details page
 
 //CNV-25427 Release note: NEW Allow VM admin to expose SSH service using LB in the UI
+* The *VirtualMachine details* -> *Details* tab now provides an option to select *SSH over LoadBalancer* when exposing a Kubernetes services. 
```

Comment:
What do you mean `exposing a Kubernetes services` here? It confuses me a bit.

Reply:
Perhaps we can remove `a Kubernetes`. [This was the article](https://docs.openshift.com/container-platform/4.12/virt/virtual_machines/vm_networking/virt-creating-service-vm.html#virt-about-services_virt-creating-service-vm) I was referring to when I chose the terminology that I did, however if it's confusing we can simply say _services_, like below:

"The *VirtualMachine details* -> *Details* tab now provides an option to select *SSH over LoadBalancer* when exposing services."

Reply:
What about?
The VirtualMachine details -> Details tab now provides a new SSH service type "SSH over LoadBalancer" to expose the SSH service over LoadBalancer.

Reply:
I changed it to what was suggested. Does that seem fine now?

Reply:
@gouyang do the changes look good at this point?